R/count_teleconnections.R

```diff
@@ -537,17 +558,21 @@ count_watershed_teleconnections <- function(data_dir,
                  n_treatment_plants,
                  watershed_pop,
                  pop_cons_m3sec,
+                 nhd_average_flow,
                  av_fl_sur_conc_pct = surface_water_concentrations[["average historical flow"]],
                  dr_fl_sur_conc_pct = surface_water_concentrations[["driest month average flow"]],
-                 av_ro_sur_conc_pct = surface_water_concentrations[["average historical runoff"]],
+                 #av_ro_sur_conc_pct = surface_water_concentrations[["average historical runoff"]],
+                 av_ro_sur_conc_pct = surface_water_concentrations[["nhd flow"]],
```

Comment:
ok I see what you've done here... but there was actually no need to develop `nhd flow`... there was already a placeholder for the flow-based concentrations in `average historical flow`, which was using the USGS gage flow that we've now supplanted with nhd.

Reply:
@swd-turner when we had our meeting last week it was mentioned that we wanted to replace the runoff values that were being used in the defacto calculations with the new NHD flow values. I took that thinking I needed to replace the "average historical runoff", but I see where I misunderstood, if you wanted to replace the USGS gage values instead, and then insert the NHD as "average historical flow".